index.html

```diff
@@ -6441,6 +6441,195 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
     -->
     </pre>
   </aside>
+
+  <section class="informative changed"><h3>Property-based data indexing</h3>
+    <p>In its simplest form (as in the examples above),
+      data indexing assigns no semantics to the keys of an <a>index map</a>.
+      However, in some situations,
+      the keys used to index objects are semantically linked to these objects,
+      and should be preserved not only syntactically, but also semantically.
+    </p>
+    <p>If the <a>processing mode</a> is set to <code>json-ld-1.1</code>,
+      <code>"@container": "@index"</code> in a term description can be accompanied with
+      an <code>"@index"</code> key. The value of that key must map to an <a>IRI</a>,
+      which identifies the semantic property linking each object to its key.
+    </p>
+    <aside class="example ds-selector-tabs"
+           title="Property-based data indexing">
+    <div class="selectors">
+      <button class="selected" data-selects="original">Original</button>
+      <button data-selects="expanded">Expanded</button>
+      <button data-selects="statements">Statements</button>
+      <button data-selects="turtle">Turtle</button>
+      <a class="playground" target="_blank"></a>
+    </div>
+    <pre class="original selected nohighlight" data-transform="updateExample">
+      <!--
+        {
+          "@context": {
+            "schema": "http://schema.org/",
+            "dc": "http://purl.org/dc/elements/1.1/",
+            "name": "schema:name",
+            "body": "schema:articleBody",
+            "words": "schema:wordCount",
+            "post": {
+              "@id": "schema:blogPost",
+              ****"@container": "@index",
+              "@index": "dc:language"****
+            }
+          },
+          "@id": "http://example.com/",
+          "@type": "schema:Blog",
+          "name": "World Financial News",
+          "post": {
+            "en": {
+              "@id": "http://example.com/posts/1/en",
```

Comment:
Probably worth adding a comment, such as the following

    ####↑ "en" will expand to `dc:language`####